monai/utils/misc.py

```diff
@@ -800,3 +801,23 @@ def _calculate_conversion_factor(self):
 
     def __call__(self, value: int | float) -> Any:
         return float(value) * self.conversion_factor
+
+
+def check_kwargs_exist_in_class_init(cls, kwargs):
+    """
+    Check if the all keys in kwargs exist in the __init__ method of the class.
+
+    Args:
+        cls: the class to check.
+        kwargs: kwargs to examine.
+
+    Returns:
+        a boolean inidicating if all keys exist.
+        a set of extra keys that are not used in the __init__.
+    """
+    init_signature = inspect.signature(cls.__init__)
```

Comment:
Perhaps this test can be simplified using https://docs.python.org/3/library/inspect.html#inspect.Signature.bind

Reply:
I am not confident using the `bind` and `partial_bind` there. And I have a feeling that some try-except may be needed to find the "extra_args". Please feel free to commit to this branch.